modules/userId/index.js

```diff
@@ -969,15 +989,26 @@ function canUseCookies(submodule) {
   return true
 }
 
+function calculateEnabledStorageTypes(submodule) {
+  const storageTypes = getConfiguredStorageTypes(submodule.config);
+
+  return storageTypes.filter(type => {
+    if (type === LOCAL_STORAGE) {
```

Comment:
Optional: Since we're selecting on equality of a single variable, you can simplify this slightly by using a switch statement instead.